src/client/Microsoft.Identity.Client/ManagedIdentity/ImdsManagedIdentitySource.cs

```diff
@@ -152,5 +139,38 @@ internal static string CreateRequestFailedMessage(HttpResponse response, string
 
             return messageBuilder.ToString();
         }
+
+        public static Uri GetValidatedEndpoint(
+            ILoggerAdapter logger,
+            string subPath,
```

Comment:
subPath is not needed to passed as a variable as it is declared as a constant above.